src/desktop/lib/preload/Electron.js

```diff
@@ -366,10 +366,10 @@ const Electron = {
         try {
             const content = await kdbx.exportVault(seeds, password);
             const now = new Date();
-
+            const prefix = seeds.length === 1 ? seeds[0].title : 'seedvault';
```

Comment:
```suggestion
            const prefix = seeds.length === 1 ? seeds[0].title : 'SeedVault';
```
I'm being really picky here 😄  would you mind making this change so that we can have consistency with mobile?

Reply:
Certainly, I can